accesshandler/pkg/providers/testvault/validate.go

```diff
@@ -0,0 +1,11 @@
+package testvault
+
+import (
+	"context"
+)
+
+// testvault validate should just run
+func (p *Provider) Validate(ctx context.Context, subject string, args []byte) error {
+
+	return nil
+}
```

Comment:
This should be removed entirely if the provider doesn't implement any validation (as per my above comments, removing the Validate() method from the Accessor interface will allow us to do this).